echopype/calibrate/env_params.py

```diff
@@ -63,13 +64,9 @@ def harmonize_env_param_time(
                 f"ping_time needs to be provided for comparison or interpolating {p.name}"
             )
```

Comment:
I wonder if the checks in Line 51-65 should be part of `align_to_ping_time`. It seems that not all cases overlap, but the check on "# If there's only 1 time1 value:" seems to be in `align_to_ping_time` already?

Reply:
So that line sends back a single value, while `align_to_ping_time` will always send back an array of the same length as the ping time DataArray receives.

Reply:
Ah I see, ok.